spec/ruby/core/class/undef_method_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../../spec_helper'
+
+describe "Class#undef_method" do
```

Comment:
There is no method `undef_method` defined on Class, it's only defined on Module.
ruby/spec is not very strict about this (e.g., there are specs for SomeClass#dup even though it should be SomeClass#initialize_copy` or so), but I think in this case it's best to have them in the same file.
I'll move the spec.